Cargo.toml

```diff
@@ -24,6 +28,14 @@ serde_json = "1.0"
 signal-hook = "0.1"
 time = "0.1"
 
+[dependencies.cpuprofiler]
+# TODO(https://github.com/AtheMathmo/cpuprofiler/pull/10): Replace this with
+# 0.0.4 or an upstream branch reference once released.
+branch = "pkg-config"
+git = "https://github.com/jmmv/cpuprofiler.git"
+optional = true
+version = "0.0.3"
```

Comment:
version doesn't do anything here. You should probably specify "rev" though.

Reply:
Oh, of course. Replaced with `rev` and removed `branch`.